### ISSUE TITLE
Zoom with tooltip disabled misplaces halo

### ISSUE THREAD

**Issue by fanciful-marmot** [highcharts/highcharts#6466]
#### Expected behaviour
The halo should disappear or be around an appropriate point.

#### Actual behaviour
The halo remains where is was before the zoom happened.

#### Live demo with steps to reproduce
http://jsfiddle.net/413pt878/

Zoom in on some points and then don't move the mouse. The halo will be on the graph but not on a point
<img width="687" alt="misplaced-halo" src="https://cloud.githubusercontent.com/assets/3170456/23903124/d1a0f60c-0899-11e7-94ab-6799ecb01a5f.png">

#### Affected browser(s)
Chrome 56, Firefox 50, Safari 10, presumably others

**Comment by pawelfus** [comment]
Hi @fanciful-marmot 

Thank you for reporting this bug. 

**Workaround:**
Demo: http://jsfiddle.net/413pt878/3/ and plugin:
```javascript
Highcharts.wrap(Highcharts.Pointer.prototype, 'drop', function(proceed) {
  proceed.apply(this, Array.prototype.slice.call(arguments, 1));

  if (this.chart.hoverPoint) {
    this.chart.hoverPoint.setState('');
  }
});
```

**Comment by pawelfus** [comment]
Fixed since v5.0.13: http://jsfiddle.net/BlackLabel/413pt878/6/
